File: Linear_search/Fortran/linear_search.f
program linear_search
  implicit none

  integer, parameter :: n = 8
  integer :: arr(n) = (/ 123, 51, 648, 999, 632, 1, 98, 10 /)
  integer :: target = 1
  integer :: i
  logical :: found

  found = .false.

  do i = 1, n
    if (arr(i) == target) then
      found = .true.
      exit
    end if
  end do

  if (found) then
    print *,   'The element ' , target, ' was found at position ', i
  else
    print *, 'The element ' , target, ' was not found'
  end if
end program linear_search

File: Linear_search/Pascal/linear_search.pas
program LinearSearch;

const
  arr: array[1..7] of Integer = (12, 1456, 53, 12451, 992, 81, 666);
  n = 7;

  var
    i, target, index: Integer;
    found: Boolean;

 begin
  target := 992;
  found := False;
  index := -1;

  for i := 1 to n do
  begin
    if arr[i] = target then
    begin
      found := True;
      index := i;
      Break;
    end;
  end;

  if found then
    WriteLn('The element ' , target, ' was found at position', index)
  else
    WriteLn('The element ', target, ' was not found');
end.